quickstarts/flutter-android/config.yml

```diff
@@ -0,0 +1,43 @@
+slug: flutter-android
+description: |
+  ## Comprehensive monitoring quickstart for Flutter Android
+  
+  Monitor your Flutter applications and analyze performance bottlenecks while operating in a live development environment. You can use this dashboard to visualize your web user experience with your application rendition.
+  
+  ## Why monitor Flutter Android?
+  Monitoring app performance enables you to take a preventative approach to improve quality, rather than waiting for issues, crashes, or user complaints. With New Relic's Android and Flutter agent, you can track network failures, slow launches, custom traces, and more.
+
+  By prioritizing active issues and marking errors as corrected, you may optimize the process once you have a comprehensive picture of all releases. Identify which release the problem first appeared in, merge duplicates, and assess whether it may get worse in the future. Add commit information to send deploy emails instantly and suggest an owner for each application error.
+  
+  ## What’s included in this quickstart?
+  New Relic's Android agent with Flutter monitoring solution captures the following metrics:
+  
+  - Be alerted on critical issues affecting your Flutter Android application
+  - Dashboards (error rate, app launches, app installations, app crashes and more)
+  - Monitor Flutter Dart exceptions and Errors
+  - Monitor HTTP response, HTTP errors and mobile events
```

Comment:
```suggestion
  - Monitor HTTP responses & errors, and mobile events
```

Reply:
I have updated it as per your comments.
Thank you...